queries.sql

```diff
@@ -16,4 +16,53 @@ SELECT * FROM animals
 WHERE name <> 'Gabumon';
 SELECT * FROM animals
 WHERE weight_kg BETWEEN 10.4 AND 17.3;
+-- Update animals table to set species as unspecified and then roll back
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals WHERE species = 'unspecified';
+ROLLBACK;
+SELECT * FROM animals WHERE species = 'unspecified';
 
+-- Update animals table to set species as digimon and then set species as pokemon
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+SELECT * FROM animals;
+
+-- Delete all records in the animals table and then roll back
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT my_savepoint;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO my_savepoint;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+SELECT * FROM animals;
```

Comment:
- [ ] As mentioned by the previous reviewer, you missed one query as seen in the project requirements screenshot below. The portion marked with the red marker is the query you have missed. 

![image](https://user-images.githubusercontent.com/84156714/229826218-5eab6707-4529-4c16-86f2-4a2482adbcb8.png)

The missed query |
--- |
![image](https://user-images.githubusercontent.com/84156714/229827430-b167c09e-d33e-4a76-8125-09205b7713fd.png)
 

The code below 👇🏾  you provided in the comment is for the query marked with the `blue` marker.

```sql
-- Delete all records in the animals table and then roll back
BEGIN;
DELETE FROM animals WHERE date_of_birth > '2022-01-01';
SAVEPOINT my_savepoint;
UPDATE animals SET weight_kg = weight_kg * -1;
ROLLBACK TO my_savepoint;
UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
COMMIT;
SELECT * FROM animals;
```

These two are different. Kindly fix this.

Reply:
If you look in description under 4. you will see:
vet_clinic=# BEGIN;
DELETE FROM animals;
ROLLBACK;
BEGIN
DELETE 11
ROLLBACK
vet_clinic=# SELECT * FROM animals;
id | name | date_of_birth | escape_attempts | neutered | weight_kg | species
----+------------+---------------+-----------------+----------+-----------+---------
13 | Plantmon | 2021-11-15 | 2 | t | -5.70 | digimon
15 | Angemon | 2005-06-12 | 1 | t | -45.00 | digimon
16 | Boarmon | 2005-06-07 | 7 | t | 20.40 | digimon
8 | Agumon | 2020-02-03 | 0 | t | 10.23 | digimon
9 | Gabumon | 2018-11-15 | 2 | t | 8.00 | digimon
11 | Devimon | 2017-05-12 | 5 | t | 11.00 | digimon
12 | Charmander | 2020-02-08 | 0 | f | -11.00 | pokemon
14 | Squirtle | 1993-04-02 | 3 | f | -12.13 | pokemon
17 | Blossom | 1998-10-13 | 3 | t | 17.00 | pokemon
18 | Ditto | 2022-05-14 | 4 | t | 22.00 | pokemon
10 | Pikachu | 2021-01-07 | 1 | f | 15.04 | pokemon
(11 rows)
And if look in my queries.sql you will see:

-- Delete all records in the animals table and then roll back
vet_clinic=# BEGIN;
DELETE FROM animals;
ROLLBACK;
BEGIN
DELETE 11
ROLLBACK